autoload/airline/extensions/nvimlsp.vim

```diff
@@ -0,0 +1,39 @@
+" Apache 2.0 license. Copyright (c) 2019-2020 Copyright Neovim contributors.
+" Plugin: https://github.com/neovim/nvim-lsp
+" vim: et ts=2 sts=2 sw=2
+
```

Comment:
You should write this config.

```viml
if !get(g:, 'nvim_lsp', 0)
  finish
endif
```

Reply:
Config was added.

Reply:
I would suggest using the same condition as in `extensions.vim`

Reply:
Changed to the condition from extensions.vim